### ISSUE TITLE
[Bug] Vec<Vec<String>> fails to build

### ISSUE THREAD

**Issue by wooden-worm** [fzyzcjy/flutter_rust_bridge#692]
### Describe the bug

I have a struct with a `Vec<Vec<String>>` field, which fails to build due to `Vec<Vec<String>>` does not implement `IntoDart`

### Codegen logs with `RUST_LOG=debug` environment variable

```Rust
struct MyData {
   data: Vec<Vec<String>>
}

pub fn test() -> anyhow::Result<MyData> {
    // code here
}
```

Output of `cargo check`

```shell
error[E0599]: the method `into_dart` exists for struct `Vec<Vec<std::string::String>>`, but its trait bounds were not satisfied
  --> src/bridge_generated.rs:70:23
   |
70 |             self.data.into_dart(),
   |                       ^^^^^^^^^ method cannot be called on `Vec<Vec<std::string::String>>` due to unsatisfied trait bounds
   |
   = note: the following trait bounds were not satisfied:
           `&Vec<Vec<std::string::String>>: Into<DartCObject>`
           which is required by `&Vec<Vec<std::string::String>>: flutter_rust_bridge::IntoDart`
           `&mut Vec<Vec<std::string::String>>: Into<DartCObject>`
           which is required by `&mut Vec<Vec<std::string::String>>: flutter_rust_bridge::IntoDart`
           `[Vec<std::string::String>]: Sized`
           which is required by `[Vec<std::string::String>]: flutter_rust_bridge::IntoDart`
           `[Vec<std::string::String>]: Into<DartCObject>`
           which is required by `[Vec<std::string::String>]: flutter_rust_bridge::IntoDart`
           `&[Vec<std::string::String>]: Into<DartCObject>`
           which is required by `&[Vec<std::string::String>]: flutter_rust_bridge::IntoDart`
           `&mut [Vec<std::string::String>]: Into<DartCObject>`
           which is required by `&mut [Vec<std::string::String>]: flutter_rust_bridge::IntoDart`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rustlib` due to previous error

```

**Comment by wooden-worm** [comment]
Hmm the errors look different. I think for this one we need something like

```rust
impl<T> IntoDartExceptPrimitive for Vec<T>
where
    T: IntoDartExceptPrimitive {}
```
in https://github.com/sunshine-protocol/allo-isolate/blob/ba1d3ee8e6d12b37d8ca63c1c13a8ff563aa5625/src/into_dart.rs#L250

**Comment by fzyzcjy** [comment]
I see. Feel free to PR!

**Comment by wooden-worm** [comment]
I created a PR here https://github.com/sunshine-protocol/allo-isolate/pull/27

**Comment by wooden-worm** [comment]
Fixed in 1.47
